swagger-backend.yaml

```diff
@@ -197,6 +209,44 @@ paths:
           description: Error
           schema:
             $ref: '#/definitions/error'
+  '/namespaces/{namespace}/devices/{device-id}/registration/namespace':
```

Comment:
I'm not entirely sure this is the right approach. Even if you wanted to go forth with it, I think it would make more sense for this to be:
`.../registration?namespace`
since you're retrieving a field/attribute of the object.
Maybe return the namespace as part of the GET or PUT return model operation? You'll save an extra call in this case :smile:

For GET:

```yaml
  device-registration-status-response:
    type: object
    properties:
      status:
        type: string
        description: >-
          Returns the device registration status, which can be one of the
          following {registered, unregistered, unknown}.
      namespace:
        type: string
        description: >-
          Returns the target namespace where the device can be found.
      message:
        type: string
        description: >-
          Exposes the error message generated at the backend when there is an
          error (example HTTP code 500).
```

Reply:
Hmmm, that's an interesting approach with using GetRegistrationStatus. Thanks!